features-json/http3.json

```
@@ -534,7 +534,8 @@
     "1":"Can be enabled in Firefox via the `network.http.http3.enabled` pref in `about:config`.",
     "2":"Can be enabled in Chrome(Chromium) via the `--enable-quic` & `--quic-version=h3-23` command line arguments.",
     "3":"Can be enabled in Chrome(Chromium) via the `--enable-quic` & `--quic-version=h3-29` command line arguments.",
-    "4":"Can be enabled in Safari on macOS 11 Big Sur and later in the \"Experimental Features\" developer menu."
+    "4":"Requires macOS 11 Big Sur or later."
```

A comma is missing to make the build happy, like so

```suggestion
    "4":"Requires macOS 11 Big Sur or later.",
```